modules/storages/spec/factories/storage_factory.rb

```diff
@@ -221,8 +222,11 @@
              refresh_token: ENV.fetch("ONE_DRIVE_TEST_OAUTH_CLIENT_REFRESH_TOKEN",
                                       "MISSING_ONE_DRIVE_TEST_OAUTH_CLIENT_REFRESH_TOKEN"),
              token_type: "bearer")
-      create(:remote_identity, oauth_client: storage.oauth_client, user: evaluator.oauth_client_token_user,
-                               origin_user_id: "33db2c84-275d-46af-afb0-c26eb786b194")
+      create(:remote_identity,
+             auth_source: storage.oauth_client,
+             user: evaluator.oauth_client_token_user,
+             integration: storage,
+             origin_user_id: "33db2c84-275d-46af-afb0-c26eb786b194")
```

Comment:
:yellow_circle: Shouldn't we be using changing values here? Otherwise all remote identities created through the factory would have colliding user names by default.

```suggestion
             origin_user_id: SecureRandom.uuid)
```

Alternatively: If there is a good reason to always use the same one, I'd go with the @mereghost approach and call it `UNIVERSALLY-DUPLICATED-IDENTIFIER`. This at least makes it transparent, that we are always dealing with the same thing and it's 5% funny ;-)

Alternatively: Just don't specify anything, because the factory for remote identities already takes care of generating sequential origin user ids.

Reply:
This origin_user_id is used in a couple of vcr cassetes. I will try to move it specifically to the specs concerned with it.